### ISSUE TITLE
keyboard mapping problem

### ISSUE THREAD

**Issue by mleduque** [cvan/keyboardevent-key-polyfill#11]
Can this be made to work with any keymap ?
For example, with my french keymap, the 48-57 range with shift seems to have completely different characters (&, é,", ', (, -, è, _, ç, à).

**Comment by joallard** [comment]
Indeed, I not only tried it with some French (<kbd>é</kbd>), but I also threw some Georgian script at it (<kbd>ტ</kbd>), it went poorly in Safari.
